frontend/appflowy_flutter/integration_test/database_cell_test.dart

```diff
@@ -268,6 +268,20 @@ void main() {
         content: DateFormat('dd/MM/y hh:mm a').format(now),
       );
 
+      await tester.tapCellInGrid(rowIndex: 0, fieldType: fieldType);
+      await tester.findDateEditor(findsOneWidget);
+
+      // Clear the date and time
+      await tester.clearDate();
+
+      await tester.dismissCellEditor();
```

Comment:
We don't need to dismiss the cell editor since clicking on the clear date button will close the popover.

Reply:
Fixed now.